src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstBase;
+    private double secondBase;
+    private double side;
+    private double height;
+
+    public IsoscelesTrapezoid(
+            Color color,
+            double firstBase,
+            double secondBase,
+            double side,
+            double height
+    ) {
+        super(color, ((firstBase + secondBase) * height) / 2);
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.